src/android/Capture.java

```diff
@@ -79,10 +77,18 @@ public class Capture extends CordovaPlugin {
     private static final int CAPTURE_PERMISSION_DENIED = 4;
     private static final int CAPTURE_NOT_SUPPORTED = 20;
 
-    private static final String[] storagePermissions = new String[]{
-        Manifest.permission.READ_EXTERNAL_STORAGE,
-        Manifest.permission.WRITE_EXTERNAL_STORAGE
-    };
+    private static String[] storagePermissions;
+    static {
+        if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            storagePermissions = new String[]{
```

Comment:
Originally thought this was a mistake, however reading through the PR I see the list is added onto on demand.

Not a merge blocker but perhaps add a comment to make this clear.

Reply:
it dont seem to work i have added all changes but on android 13 it gives permission denied error